### PR TITLE
Porting PR 917

### PR DESCRIPTION
**Pull request by swati-sh**
Resolve merge conflict

### CODE REVIEW

File: core/jazz_ui/src/app/core/services/admin-utils.service.ts [hunk 1]
```diff
@@ -1,12 +1,11 @@
-import {Injectable} from '@angular/core';
-import {Http} from "@angular/http";
-import {Observable} from "rxjs/Observable";
-import {RequestService} from "./request.service";
+import { Injectable } from '@angular/core';
+import { Http } from "@angular/http";
+import { Observable } from "rxjs/Observable";
+import { RequestService } from "./request.service";
```
Comment (raghits) [inline core/jazz_ui/src/app/core/services/admin-utils.service.ts]: can we be consistent, probably without spaces like {Observable}

File: core/jazz_ui/src/app/pages/admin-dashboard/admin-dashboard.component.ts [hunk 2]
```diff
@@ -1,5 +1,7 @@
 import {AfterViewChecked, AfterViewInit, Component, OnInit, ViewChild} from '@angular/core';
+import { ToasterService} from 'angular2-toaster';
```
Comment (raghits) [inline core/jazz_ui/src/app/pages/admin-dashboard/admin-dashboard.component.ts]: inconsistent, see comment above.

File: core/jazz_ui/src/app/core/services/admin-utils.service.ts (hunk 1 above)
Comment (swati-sh) [reply core/jazz_ui/src/app/core/services/admin-utils.service.ts]: Done the changes.

File: core/jazz_ui/src/app/pages/admin-dashboard/admin-dashboard.component.ts (hunk 2 above)
Comment (swati-sh) [reply core/jazz_ui/src/app/pages/admin-dashboard/admin-dashboard.component.ts]: done the changes.

**Pull request merged**